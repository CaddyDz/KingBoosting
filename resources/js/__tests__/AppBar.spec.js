import {
    shallowMount
} from '@vue/test-utils';

import AppBar from '../components/AppBarComponent.vue';

describe('AppBar', () => {
    it('renders', () => {
        let wrapper = shallowMount(AppBar, {
            stubs: ['nav-links', 'LoginComponent']
        });
        expect(wrapper).toMatchSnapshot();
    });
});
