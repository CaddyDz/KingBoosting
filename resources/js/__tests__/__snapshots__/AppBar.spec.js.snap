// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar renders 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
